examples/with-phone-password/README.md

```diff
@@ -46,27 +46,31 @@ This demo app uses the EmailPassword and Passwordless recipes to achieve the aut
 
 ### Frontend
 
+-   Add a custom `PhoneVerifiedClaim`.
+-   Override `getGlobalClaimValidators` in the Session recipe to add a validator for `PhoneVerifiedClaim`. This will add an error into `invalidClaims` in the session context of components wrapped by `SessionAuth` if PhoneVerifiedClaim` is missing or set to false.
```

Comment:
```suggestion
-   Override `getGlobalClaimValidators` in the Session recipe to add a validator for `PhoneVerifiedClaim`. This will add an error into `invalidClaims` in the session context of components wrapped by `SessionAuth` if `PhoneVerifiedClaim` is missing or set to false.
```